tests/integration/basic_data/policy-pack/PulumiPolicy.yaml

```diff
@@ -0,0 +1,2 @@
+description: A Policy Pack to test certain data is available to policy packs
```

Comment:
Nit: Should this end with a period? IMHO, it looks like it should.

Reply:
"certain data" isn't especially useful. (Similarly `basic_data` is equally vague.)

What do you think about calling the test "runtime_data" or "pulumi_sdk"?

Reply:
I dont think we typically have this end in a period, but agree that follows suit to our other comments/text throughout our repos. 

Should it be `available to Policy Packs`?